### PR TITLE
Feature/140/meet the team page

### PR DESCRIPTION
**Pull request by rbrodefrank**
### Describe the problem being solved:
Fixed text styling on Meet the Team page to match styling themes of home page.

Before:
feature-140-css-before

After:
feature-140-css-after

### Impacted areas in the application: 
Meet the Team page

List general components of the application that this PR will affect: 
* frontend/src/components/static_pages/team.jsx
* frontend/src/components/static_pages/team.css

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts
- [x] I have run the [prettier](https://prettier.io/) command `make pretty`

### CODE REVIEW

File: frontend/src/components/static_pages/team.css [hunk 1]
```diff
@@ -3,8 +3,15 @@
   font-size: 1.2rem;
 }
 
-.meetTheTeam h2 {
-  margin-bottom: 20px;
+.heading {
+  font-weight: bold;
+  font-size: 36pt;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/team.css]: i know she gave you these in pt but can we use pixels -> should be 48 

also this class is used on the Home Page -> lower mid but i think we want to make the change there too

File: frontend/src/components/static_pages/team.css [hunk 2]
```diff
@@ -3,8 +3,15 @@
...
+  padding: 15px 0;
+  margin-top: 25px;
+}
+
+.line-paragraph {
+  font-size: 24pt;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/team.css]: 32px

File: frontend/src/components/static_pages/team.css [hunk 3]
```diff
@@ -58,6 +65,14 @@
   .teamPhotos .facebookLinkContainer p {
     font-size: medium;
   }
+  .heading {
+    font-size: 18pt;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/team.css]: 24px

File: frontend/src/components/static_pages/team.css [hunk 4]
```diff
@@ -67,4 +82,15 @@
     height: 200px;
     width: 250px;
   }
+
+  .heading {
+    font-size: 26pt;
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/team.css]: 35px but i think we want 36px

File: frontend/src/components/static_pages/team.css (hunk 1 above)
Comment (rbrodefrank) [reply frontend/src/components/static_pages/team.css]: Michele specifically asked for all of the styling to be the same so I assumed that we should use the same class name, is that a bad practice that should be changed?

Comment (dkaplan88) [reply frontend/src/components/static_pages/team.css]: Yeah thats why I think its fine. Just not sure if they realized the implications 👍 

Def best practice to have constant styles from page to page -> I am a big fan of the SMACSS Framework http://smacss.com/

**Pull request merged**